fizz/record/RecordLayer.cpp

```diff
@@ -89,10 +89,10 @@ Param parse<ServerHello>(Buf handshakeMsg, Buf original) {
     hrr.extensions = std::move(shlo.extensions);
 
     hrr.originalEncoding = std::move(original);
-    return std::move(hrr);
+    return hrr;
```

Comment:
You cannot elide the move here. NRVO doesn't apply since there are two branches that return different types. We need the `std::move()` here to explicitly invoke the variant constructor.

Reply:
Full output: https://gist.github.com/neheb/61312a8f1a30ae1168f07cb2f7929c49